openlibrary/templates/lib/nav_foot.html

```diff
@@ -8,6 +8,7 @@ <h2>Open Library</h2>
         <ul>
           <li><a href="/about/vision">$_('Vision')</a></li>
           <li><a href="/volunteer">$_('Volunteer')</a></li>
+          <li><a href="/partner-with-us" title="$_('Partner With Us')">$_('Partner With Us')</a></li>
```

Comment:
Don't duplicate title if it's same as link content.
```suggestion
          <li><a href="/partner-with-us">$_('Partner With Us')</a></li>
```